src/Caller/Masa.Utils.Caller.Core/DefaultRequestIdGenerator.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Utils.Caller.Core;
+
+public class DefaultRequestIdGenerator : IRequestIdGenerator
+{
+    public string NewId() => Guid.NewGuid().ToString();
```

Comment:
Why you don't use IdGenerator?

Reply:
Later migration to BuildingBlock will adjust